scripts/jenkins/cloud/manual/deployer-entry-point.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -i
+
+: ${git_user_name:=Unknown User}
+: ${git_user_email:=unknown@unknown.org}
+: ${python:=python2}
+
+configure_git() {
+    git config --global user.name "${git_user_name}"
+    git config --global user.email "${git_user_email}"
+}
+
+set_up_venv() {
+    ln -s "/opt/ansible-venv-${python}" /opt/ansible-venv
```

Comment:
The ln comment will also fail if this is deployer-entry-point.sh is run more than once. 
Perhaps a nicer logic to handle the ln already existing. not important but could prevent problems in the future. 
something like this: 
```
set_up_venv() {
    filelink='/opt/ansible-venv'
    if [ ! -e $filelink ]; then
       ln -s "/opt/ansible-venv-${python}" $filelink
    fi
}
```

Reply:
Combining the two comments. it could look something like: 
```
set_up_venv() {
    filelink='/opt/ansible-venv'
    if [ ! -e $filelink ]; then
       ln -s "/opt/ansible-venv-${python}" $filelink
    fi
    export ANSIBLE_STRATEGY_PLUGINS=$(find /opt/ansible-venv/lib64/*/site-packages/ansible_mitogen/plugins -name strategy)
}

```